modules/ROOT/pages/api-analytics-dashboard.adoc

```diff
@@ -29,13 +29,150 @@ image::api-analytics-detail-panel.png[]
 +
 You can click the *View API in API Manager* link to view API analytics in API Manager.
 
-== Requests
+== New Metrics View for API Analytics Dashboards
+
+API analytics dashboards include various pages to view different types of charts:
+
+[%header,cols="1,4"]
+|===
+| API Analytics Dashboards | Charts
+
+| xref:api-analytics.adoc#overview-charts[Overview] a|
+
+* Request Volume
+* Failed Requests
+* Policy Violations
+* Average Response Time
+
+| xref:api-analytics.adoc#requests-charts[Requests] a|
+
+* Requests
+* Requests by Method
+* Requests by Response Code
+* Requests by Request Size
+* Requests by Response Size
+
+| xref:api-analytics.adoc#policies-charts[Policies] a|
+
+* Requests
+* Policy Violations by Outcome
+* Policy Violations by Violation ID
+
+| xref:api-analytics.adoc#failures-charts[Failures] a|
+
+* Failures
+* Failures by Method
+* Requests by Response Code
+* Requests by Request Size
+* Requests by Response Size
+
+| xref:api-analytics.adoc#performance-charts[Performance] a|
+
+* Requests by Performance (line chart)
+* Requests by Performance (table)
+* Slow Requests (requires Titanium)
+
+| xref:api-analytics.adoc#client-applications-charts[Client Applications] a|
+
+* Requests by Client ID
+* Requests by Client IP
+
+|===
+
+[[overview-charts]]
+=== Overview Charts
+
+The charts on the *Overview* dashboard are defined as follows:
```

Comment:
active? Like The following charts appear on the ....

Reply:
```suggestion
The following charts appear on the *Overview* dashboard:
```